routes/items.js

```diff
@@ -28,11 +28,10 @@ router.get('/', async (req, res) => {
   const { search, type, rating, status, location } = req.query
 
   const usersInLocation = location ? (await User.find({ location })).map((user) => user._id) : undefined
-
   const query = {
-    ...(search ? { name: { $regex: new RegExp(search, 'i') } } : {}),
+    ...(search ? {$or:[{name:{ $regex: new RegExp( "^"+search, 'i') }},{name:{ $regex: new RegExp( " "+search, 'i') }}]} : {}),
```

Comment:
Remember to use the linter on both the frontend and backend files.

Reply:
will look into it, thanks